operations/package_repository_management/Troubleshoot_Nexus.md

```diff
@@ -78,3 +79,14 @@ successful. If `helm status -n nexus cray-nexus` indicates the status is
 deployment and additional diagnosis is required. In this case, the current
 Nexus deployment probably needs to be uninstalled and the `nexus-data` PVC
 removed before attempting to deploy again.
+
+## Unable to login with Keycloak accounts
+
+First ensure the Keycloak account that is attempting to login has proper Nexus permissions.
+See [Manage Repositories with Nexus](./Manage_Repositories_with_Nexus.md) for information
```

Comment:
./ is superfluous in this context. In scripts it makes sense, because of PATH issues, but not in a link context.
```suggestion
See [Manage Repositories with Nexus](Manage_Repositories_with_Nexus.md) for information
```